data/phi0474/phi001/phi0474.phi001.perseus-eng2.xml

```diff
@@ -41,16 +39,18 @@
       <sourceDesc>
         <biblStruct>
           <monogr>
-            <author>M. Tullius Cicero</author>
-            <title>The Orations of Marcus Tullius Cicero, literally translated by C. D.
-              Yonge</title><idno type="OCLC">4709897</idno>
+            <author>Cicero</author>
```

Comment:
I reread your comments this morning on Claudian after submitting this pull request and I actually do get what you meant there, about how the name shows up in the book. In the Claudian book they list Claudian so I left it that way. I hadn't looked at the printed Cicero texts but will start looking at all scanned texts before I change names.

Reply:
@AlisonBabeu 
I think all of the Latin and all of the English are from only a couple of editions/series. So it would probably be best to just pick one for each of these types of items (top level author, monograph author) apply it en masse for the Cicero corpus.
The irony being that we are still stuck with an unsophisticated TOC in the reading environment, so that none of this helps in the context of the Scaife Viewer. (sigh)

Reply:
In other words, the consistency is more important than reproducing precisely what each print volume calls him. I'd rather have all of the files use the same darn `<author>` name in the same order and with the same abbreviation formulation here.